runtime/cudaq/distributed/builtin/activate_custom_mpi.sh

```diff
@@ -48,7 +48,9 @@ fi
 
 echo "Using $CXX to build the MPI plugin for MPI installation in $MPI_PATH."
 lib_mpi_plugin="$this_file_dir/libcudaq_distributed_interface_mpi.so"
+# --disable-mlir-links for https://github.com/NVIDIA/cuda-quantum/issues/2892
```

Comment:
It's ok to put this sort of documentation (cross referencing) in the commit message rather than in the source code.

Reply:
Since this is a shell script that gets shipped with the container, I felt like it would be useful having it in the script itself so users could reference it more easily.

Reply:
I don't think that is our process in general.

Reply:
I removed the comment in the latest commit.